styleguide_example/files/tests/flows/test_standard_upload.py

```diff
@@ -1,13 +1,93 @@
+from django.conf import settings
 from django.test import TestCase
+from django.urls import reverse
+from django.core.files.uploadedfile import SimpleUploadedFile
+
+from rest_framework.test import APIClient
+
+from styleguide_example.files.models import File
+from styleguide_example.files.utils import file_generate_content
+from styleguide_example.users.models import BaseUser
+from styleguide_example.users.services import user_create
 
 
 class StandardUploadApiTests(TestCase):
     """
     We want to test the following general cases:
 
     1. Upload a file, below the size limit, assert models gets created accordingly.
-    1. Upload a file, above the size limit (patch settings), assert API error, nothing gets created.
+    2. Upload a file, above the size limit (patch settings), assert API error, nothing gets created.
+    3. Upload a file, equal to the size limit, assert models gets created accordingly
     """
+    def setUp(self):
+        self.client = APIClient()
+
+        self.jwt_login_url = reverse("api:authentication:jwt:login")
+        self.standard_upload_url = reverse("api:files:upload:standard")
+
+    def test_standard_upload(self):
+        client = self.client
+
+        self.assertEqual(0, File.objects.count())
+        self.assertEqual(0, BaseUser.objects.count())
+
+        # Create a user
+        credentials = {
+            "email": "some_email@hacksoft.io",
+            "password": "123456"
+        }
+        user_create(**credentials)
+        self.assertEqual(1, BaseUser.objects.count())
+
+        # Log in and get the authorization data needed
+        response = client.post(self.jwt_login_url, credentials)
+        self.assertEqual(200, response.status_code)
+        data = response.data
+        token = data["token"]
+        auth_headers = {
+            "HTTP_AUTHORIZATION": f"{settings.JWT_AUTH['JWT_AUTH_HEADER_PREFIX']} {token}"
+        }
+
+        # Create a small sized file
+        file_1 = SimpleUploadedFile(
+            name="file_small.txt",
+            content=b"file_content_test_1",
+            content_type="text/plain"
+        )
+
+        with self.subTest("1. Upload a file, below the size limit, assert models gets created accordingly"):
+            response = client.post(
+                self.standard_upload_url, {"file": file_1}, enctype="multipart/form-data", **auth_headers
+            )
+
+            self.assertEqual(201, response.status_code)
+            self.assertEqual(1, File.objects.count())
+
+        # Create a file above the size limit
+        file_2 = SimpleUploadedFile(
+            name="file_big.txt", content=file_generate_content(k=10485761), content_type="text/plain"
```

Comment:
Large strings can be generated by multiplying a single character.
Let's do that instead of exporting a new utility function.